lib/pay.rb

```diff
@@ -61,6 +61,7 @@ module Billable
   @@emails.subscription_renewing = ->(pay_subscription, price) {
     (price&.type == "recurring") && (price.recurring&.interval == "year")
   }
+  @@emails.subscription_trial_will_end = true
```

Comment:
Do we need another setting for `subscription_trial_ended`?

Reply:
Oh yeah good call, probably so.